appimage-amd64.yml

```diff
@@ -0,0 +1,78 @@
+version: 1
+
+script:
+  - mkdir -p build
+  - cd build; cmake .. -DCMAKE_INSTALL_PREFIX=/usr
+  - cd build; make -j`nproc` install DESTDIR=AppDir
```

Comment:
Probably can be `cmake --build build --target install`, but not important.

Reply:
Yes, Use the method you prefer. Just make sure it gets installed into ./AppDir.